qrisk/stats.py

```diff
@@ -166,67 +149,81 @@ def max_drawdown(returns):
 
     df_cum_rets = cum_returns(returns, starting_value=100)
     cum_max_return = df_cum_rets.cummax()
+    drawdown = df_cum_rets.sub(cum_max_return).div(cum_max_return)
 
-    return df_cum_rets.sub(cum_max_return).div(cum_max_return).min()
+    return drawdown.min()
 
 
-def annual_return(returns, period=DAILY):
-    """Determines the annual returns of a strategy.
+def annual_return(returns, period=DAILY, annualization=ANNUALIZATION_FACTORS):
```

Comment:
Futures have different number of business days in a year than equities. Sometimes it is dependent on the commodity at hand. This is part of the API so that qrisk is futures compatible.